question-servers/freeform.js

```diff
@@ -845,7 +845,7 @@ module.exports = {
                 cache.get(cacheKey, (err, cachedData) => {
                     // We don't actually want to fail if the cache has an error; we'll
                     // just render the panel as normal
-                    ERR(err, (e) => logger.error(e));
+                    ERR(err, (e) => logger.error('Error in cache.get()', e));
```

Comment:
```suggestion
                    ERR(err, (e) => logger.error('Error in get()', e));
```
Not sure if intentional? `get()` is too generic I guess?

Reply:
Yep, `get()` just felt super-generic. The exact error string doesn't really matter here too much because it's the stack trace that we really want.